app/views/email_authentications/invitations/edit.html.erb

```diff
@@ -1,3 +1,6 @@
+<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```

Comment:
Is this the full bootstrap CSS?  Or a limited set that you just need for this...we already have bootstrap in the app wide bundle.

Reply:
I don't know, but the eye asset it's looking for does not appear to be accessible without adding it. I'd speculate there's a better way of handling this, but the jquery function does hard code the icon name.